### DISCUSSION TITLE
how to process result in auto api

### DISCUSSION THREAD

**Discussion by One-One-Code** [abpframework/abp#18143] [Q&A]
In application service return dto ,and map api with auto api,

I want process all api result with standard model ，anyone can tell me how to solved it？

AppService like below：
![image](https://github.com/abpframework/abp/assets/44698989/7771f7ed-7316-413f-be2e-539a11eee0ee)

I want return in api like：
WebApiResult<List<AddressDto>>

WebApiResult is my custom standard model

**Comment by maliming** [comment]
The simplest way is using `WebApiResult` as the return type.

**Reply by One-One-Code** [reply]
i want to  process it by interceptor，but i cannot find any sample,
